### DISCUSSION TITLE
Name colision in crates.io

### DISCUSSION THREAD

**Discussion by dev7ba** [antonilol/rust-bitcoincore-zmq#2] [General]
Hi, antonilol.
:-)

I was about to upload my version of bitcoincore-zmq 
https://github.com/dev7ba/bitcoincore-zmq
And I've realized that you took that name 2 months before!

I really don't care, your version also includes support for other msg rather than sequence msgs... mine not. So better for the community.

I created mempoolexplorer.com a year ago and I was wondering what are you involved in now. Just to avoid other collisions and maybe finding synergies with your work. I haven't found any other place to contact you but github, feel free to delete this issue.

You can contact me by any means described in the footpage at mempoolexplorer.com.

See you.

Dev7ba.

**Comment by antonilol** [comment]
> I really don't care, your version also includes support for other msg rather than sequence msgs... mine not. So better for the community.

you could consider using my bitcoincore-zmq, feel free to open issues/prs for any missing features

> I created mempoolexplorer.com a year ago and I was wondering what are you involved in now. Just to avoid other collisions and maybe finding synergies with your work.

i work on a couple projects of my own (see my github, most recently this repo and bitcoin-script-analyzer), and also contributed (and still do) to various other projects like mempool.space and sphinx-relay
